### PR TITLE
typehint $app in routes.php and pipeline.php #154

### PR DESCRIPTION
**Pull request by belgattitude**

### CODE REVIEW

**Comment by belgattitude** [comment]
Looks coveralls check is too strict, was only 2 comments added.

File: src/ExpressiveInstaller/Resources/config/routes-full.php [hunk 1]
```diff
@@ -24,6 +24,8 @@
  *     Zend\Expressive\Router\Route::HTTP_METHOD_ANY,
  *     'contact'
  * );
+ *
+ * @var \Zend\Expressive\Application $app
```
Comment (xtreamwayz) [inline src/ExpressiveInstaller/Resources/config/routes-full.php]: Somehow this feels weird. That docblock is a page level docblock.  Wouldn't it be better to write in a separate line like this: 

` /** @var \Zend\Expressive\Application $app */`

Comment (belgattitude) [inline src/ExpressiveInstaller/Resources/config/routes-full.php]: Yes you're right, I've updated the P/R to separate the docblock. Thanks

**Comment by belgattitude** [comment]
Updated P/R to separate the docblock as suggested by @xtreamwayz 

```php
/** @var \Zend\Expressive\Application $app */
```

**Comment by xtreamwayz** [comment]
Merged, thanx @belgattitude

**Pull request closed without merging**